SetupWizard/ImportSettingsPage.cs

```diff
@@ -562,23 +584,35 @@ private void MergeSettings(string settingsFilePath)
 						string layerHeight;
 
 						bool isSlic3r = settingsToImport.TryGetValue(SettingsKey.layer_height, out layerHeight);
+						bool containsValidSetting = false;
 						//if (isSlic3r)
 						{
 							var activeSettings = ActiveSliceSettings.Instance;
 
 							foreach (var item in settingsToImport)
 							{
-								// Compare the value to import to the layer cascade value and only set if different
-								string currentValue = activeSettings.GetValue(item.Key, null).Trim();
-								if (currentValue != item.Value)
+								if(activeSettings.Contains(item.Key))
 								{
-									activeSettings.UserLayer[item.Key] = item.Value;
+									containsValidSetting = true;
+									string currentValue = activeSettings.GetValue(item.Key, null).Trim();
```

Comment:
Yes it does default to null I didn't actually edit that line it is merely tabbed in due to be in an additional If statement. I can remove it if needed.